collections/_article/review-godotcon24.md

```diff
@@ -0,0 +1,164 @@
+</div>
+<div class="preview-image-container">
+	<img alt="Keep The Heroes Out booth" src="/storage/blog/godotcon24/keep-out.webp"/>
+	<img alt="Looking Up I See Only A Ceiling booth" src="/storage/blog/godotcon24/looking-up.webp"/>
+	<img alt="PVKK: Planetenverteidigungskanonenkommandant booth" src="/storage/blog/godotcon24/pvkk.webp"/>
+	<img alt="Robot Detour booth" src="/storage/blog/godotcon24/robot-detour.webp"/>
+	<img alt="Voice of Flowers booth" src="/storage/blog/godotcon24/voice-of-flowers.webp"/>
+	<img alt="Window Arcade booth" src="/storage/blog/godotcon24/window-arcade.webp"/>
+</div>
+
+**Exhibitor list:**
+- [Bug Off](https://curio-compass-games.itch.io/bug-off)
+- [Data Garden](https://store.steampowered.com/app/2644670/Data_Garden/)
+- [Dung Slinger](https://s4g.itch.io/dung-slinger)
+- [Fogpiercer](https://store.steampowered.com/app/3219010/Fogpiercer/)
+- [Immersive Home](https://immersive-home.org/)
+- [In Good Faith](https://store.steampowered.com/app/3151800/In_Good_Faith/)
+- [Keep The Heroes Out](https://store.steampowered.com/app/2592340/Keep_the_Heroes_Out/)
+- [Looking Up I See Only A Ceiling](https://store.steampowered.com/app/1742930/Looking_Up_I_See_Only_A_Ceiling/)
+- [PVKK: Planetenverteidigungskanonenkommandant](https://store.steampowered.com/app/2956040/PVKK_Planetenverteidigungskanonenkommandant/)
+- [Robot Detour](https://store.steampowered.com/app/2666840/Robot_Detour/)
+- [Skye's Ducky Adventure: Quest for Gluttony](https://schwarzeralptraum.itch.io/skyes-ducky-adventure-quest-for-gluttony)
+- [Voice of Flowers](https://store.steampowered.com/app/2609560/Voice_of_Flowers/)
+- Window Arcade
+
+# Activities
+
+<img alt="The graffiti canvas full of different drawings by the visitors" src="/storage/blog/godotcon24/canvas.webp"/>
+
+What do you do at a conference, besides watching talks, browsing through the exhibition, or doing business? Have some fun and get to know other attendees in a casual way, of course.
+
+This is exactly what we aimed to facilitate with our onsite activities, especially the [sticker challenge](https://conference.godotengine.org/2024/activities/) tying them all together (since many of you asked: the stickers were designed by community artist [Tibo](https://gotibo.fr/)). From networking bingo to graffiti canvas, we saw many people striking up conversations with strangers or having a great time with friends.
+
+<img alt="People participating in a game jam" src="/storage/blog/godotcon24/one-pc-jam.webp"/>
+
+One of the most popular activities turned out to be the One PC Jam (taking place at three laptops concurrently) where the conference attendees created a jam game over the course of two days. The twist: everyone only was granted one hour on the project, before having to concede their seat to the next person. This resulted in three separate, actually playable games, which you can [find here](https://godotcon.itch.io/).
```

Comment:
```suggestion
One of the most popular activities turned out to be the One PC Jam (taking place at three laptops concurrently) where the conference attendees created a jam game over the course of two days. The twist: everyone only was granted one hour on the project, before having to concede their seat to the next person. This resulted in three separate, actually playable games, which you can [find on our GodotCon itch.io site](https://godotcon.itch.io/).
```
Generally best to avoid "here" in links.